extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -23,7 +23,7 @@ const {
 const BackbeatProducer = require('../../lib/BackbeatProducer');
 const locations = require('../../conf/locationConfig.json') || {};
 
-const nSecsPerDay = () => 24 * 60 * 60 / config.timeOptions.timeProgressionFactor;
+const nSecsPerDay = () => Math.ceil(24 * 60 * 60 / config.timeOptions.timeProgressionFactor);
```

Comment:
You could potentially leverage the `scaleMsPerDay` utility function: https://github.com/scality/Arsenal/blob/development/8.1/lib/s3middleware/objectUtils.ts#L20